### DISCUSSION TITLE
How to log in to LNbits with user/password? OS permission issue?

### DISCUSSION THREAD

**Discussion by adevoss** [raspiblitz/raspiblitz#4857] [Q&A] [answered]
Hello LNbits users,

After creating a user
When I log in I get: `Invalid credentials 401`
When I log in by pasting the admin url the `Save` button on the `Server` page is disabled.

Is this a Raspiblitz or LNbits issue?

(v1.11.3-albyhub / 0.12.11)

**Comment by rootzoll** [comment]
"Creating a User" you mean after a fresh install when creating the admin user thru the webui?

Hav you tried to deinstall (delete database - when no critical data yet) and reinstall? Can you screenshot errors?

**Comment by adevoss** [comment, marked as answer]
Created a normal user in LNbits.
Can't log in at http://local-ip:5001 with username/password. When I log in with User ID I can log in.
Update: Changed password (again) from my pasword manager. Now I can log in.

Reinstall LNbits is no option as it is my live node and there are enough funds not to want to loose them.

So I don't really understand LNBits yet. Why the 'Save' button is disabled I don't know. But everything is fine for now.
Thanks for replying.

Background: Trying to self-host a LNaddress at my own domain.
